dht_bootstrap.go

```diff
@@ -88,14 +144,52 @@ func (dht *IpfsDHT) startRefreshing() error {
 		}
 	})
 
+	// when our address changes, we should proactively tell our closest peers about it so
+	// we become discoverable quickly. The Identify protocol will push a signed peer record
+	// with our new address to all peers we are connected to. However, we might not necessarily be connected
+	// to our closet peers & so in the true spirit of Zen, searching for ourself in the network really is the best way
+	// to to forge connections with those matter.
+	dht.proc.Go(func(proc process.Process) {
+		for {
+			select {
+			case evt, more := <-dht.subscriptions.everPeerAddressChanged.Out():
+				if !more {
+					return
+				}
+				switch evt.(type) {
+				case event.EvtLocalAddressesUpdated:
+					// our address has changed, trigger a self walk so our closet peers know about it
+					res := make(chan error, 1)
```

Comment:
I believe we can send a nil channel if we don't want a response (but double check that).

Reply:
We can indeed. This is done.